_events/2022-All-Hands-Meeting.md

```diff
@@ -56,12 +56,12 @@ Hybrid event both at the University of Oklahoma (OU) in Norman and virtual.
 
 ### Fees
 
-In-person participation will include a standard registration fee; online-only participation will continue to be free. 
+In-person participation will include a $70 registration fee; online-only participation will continue to be free. 
 
 
 ### Registration
 
-Required registration for both components will open in January 2022.
+Required registration for both components. Visit [here](https://opensciencegrid.org/all-hands/2022/registration/) for complete event and registration information. Note: the in-person registration has a deadline of February 17. 
```

Comment:
But here, since the text reads “for complete event … information”, I would point to the 2022 home page.